pkg/utils/taints.go

```diff
@@ -0,0 +1,110 @@
+package utils
+
+// Copy paste from SNR - https://github.com/medik8s/self-node-remediation/blob/main/pkg/utils/taints.go
+import (
+	"context"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	"github.com/medik8s/fence-agents-remediation/api/v1alpha1"
+)
+
+var (
+	loggerTaint = ctrl.Log.WithName("taints")
+)
+
+// taintExists checks if the given taint exists in list of taints. Returns true if exists false otherwise.
+func taintExists(taints []corev1.Taint, taintToFind *corev1.Taint) bool {
+	for _, taint := range taints {
+		if taint.MatchTaint(taintToFind) {
```

Comment:
Heads up, this doesn't check for equal taint value!
Maybe it's a good idea to use unique taint keys per operator.

Reply:
Or check for taint's value as well 👐🏻

Reply:
>Maybe it's a good idea to use unique taint keys per operator.

I agree, and the `MatchTaint` func declaration supports that:
> // MatchTaint checks if the taint matches taintToMatch. Taints are unique by key:effect,
// if the two taints have same key:effect, regard as they match.

How about `"medik8s.io/far-remediation"` for FAR, and probably SNR should change it's key to include it's name, so maybe `"medik8s.io/snr-remediation"`.
CC: @mshitrit

Reply:
Following this understanding and [previous review](https://github.com/medik8s/fence-agents-remediation/pull/53#discussion_r1226239684) do you still support the decision to have a value for the taint? I am less pro for having for the value now, but I don't have a strong preference.